crates/mpz-garble-core/Cargo.toml

```diff
@@ -9,9 +9,6 @@ workspace = true
 [lib]
 name = "mpz_garble_core"
 
-[profile.release]
-lto = true
-
```

Comment:
Nor sure what was the reason here, so pointing at this specifically. I removed it since it is ignored anyways when building the workspace. I assumed we do not want full LTO support on the whole workspace?

Reply:
I am getting this when doing `cargo check`
```
warning: profiles for the non root package will be ignored, specify profiles at the workspace root:
package:   /home/th4s/tlsnotary/mpz/crates/mpz-garble-core/Cargo.toml
workspace: /home/th4s/tlsnotary/mpz/Cargo.toml

```